### PR TITLE
Create a githubteacher hometown file

### LINKED ISSUE #14: GitHubTeacher's recommendations for Fuquay Varina, NC
# Fuquay- Varina, NC
- [x] Create a branch
- [x] Add a file
- [x] Commit the file
- [x] Open a Pull Request
- [x] Collaborate with my team (YOU!)
- [x] Get a :+1:
- [x] :shipit: Merge the Pull Request

@kakuls @hollenberry anything I forgot?

### PR DESCRIPTION
**Pull request by githubteacher**
Resolves #14 
## What do you want to know?
- Do you like any specific types of food?
- Any types of activities you enjoy?

@dmgross144 @hollenberry @kakuls

### CODE REVIEW

**Comment by hollenberry** [comment]
@githubteacher 

> Do you like any specific types of food?

I like Sushi! Know any good places?

> Any types of activities you enjoy?

Yodeling contests.

**Comment by dmgross144** [comment]
@githubteacher 

> Do you like any specific types of food?
> I like brats with sauerkraut and bier
> 
> Any types of activities you enjoy?
> mountain biking and hiking the Colorado Trail

File: places/githubteacher-hometown.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Fuquay-Varina, NC
+
+### Where to eat
+
+- The best sushi is in Durham at Blue Sushi
```
Comment (githubteacher) [inline places/githubteacher-hometown.md:5]: Where is this fabled place?

**Comment by bmdangol** [comment]
Looks good. 👍

**Pull request merged**